golang/internal/grpc/grpc.go

```diff
@@ -176,14 +182,40 @@ func Init(grpcContext context.Context,
 		grpcConn.Conn = conn
 	}
 
-	grpcLoop(ctx, connParams.nodeID, workerFuncs, cancel, appConfig)
+	grpcLoop(ctx, connParams, workerFuncs, cancel, appConfig)
+}
+
+func grpcProcessCommand(
+	ctx context.Context,
+	workerFuncs WorkerFunctions,
+	command *agent.AgentCommand,
+	appConfig *config.CommonConfiguration,
+) {
+	if command.GetDeploy() != nil {
```

Comment:
Don't we want to change it to a switch case?

Reply:
:o I didn't know golang switch-cases could replace something like this.